### PR TITLE
Fix a typo in config comment

### LINKED ISSUE #7664: neo4j.conf incorrect documentation for LOAD CSV setting
This documentation in the configuration file is incorrect: 

```
# This setting constrains all `LOAD CSV` import files to be under the `import` directory. Remove or uncomment it to
# allow files to be loaded from anywhere in filesystem; this introduces possible security problems. See the `LOAD CSV`
# section of the manual for details.
dbms.directories.import=import
```

It should say `Remove or comment`

**Neo4j Version:** 3.0.3  
**Operating System:** Mac OSX El Capitan

### PR DESCRIPTION
**Pull request by spacecowboy**
fixes #7664

### CODE REVIEW

File: packaging/standalone/standalone-community/src/main/distribution/text/community/UPGRADE.txt [hunk 1]
```diff
@@ -1 +1 @@
-For upgrade instructions, please see http://neo4j.com/docs/operations-manual/current/#upgrading.
+For upgrade instructions, please see http://neo4j.com/docs/operations-manual/current/.
```
Comment (Mats-SX) [inline packaging/standalone/standalone-community/src/main/distribution/text/community/UPGRADE.txt]: I think this is the new link: http://neo4j.com/docs/operations-manual/current/deployment/#upgrade

File: packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/UPGRADE.txt [hunk 2]
```diff
@@ -1 +1 @@
-For upgrade instructions, please see http://neo4j.com/docs/operations-manual/current/#upgrading.
+For upgrade instructions, please see http://neo4j.com/docs/operations-manual/current/.
```
Comment (Mats-SX) [inline packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/UPGRADE.txt]: Same here; http://neo4j.com/docs/operations-manual/current/deployment/#upgrade

**Comment by spacecowboy** [comment]
@Mats-SX corrected the links

**Comment by spacecowboy** [comment]
Already fixed.

**Pull request closed without merging**